app/views/docs/database.phtml

```diff
@@ -1,22 +1,35 @@
 <p>
     The Database Service allows you to store your application and users' data and fetch it using different supported queries.
-    Using your Appwrite Database, you can organize your data into collections and documents using the Appwrite simple REST API. You can also use the Appwrite <a href="/docs/realtime">Realtime API</a> to subscribe to live changes in your collections and documents.
+    Using your Appwrite Database, you can create multiple databases, organize your data into collections and documents using the Appwrite REST API. You can also use the Appwrite <a href="/docs/realtime">Realtime API</a> to subscribe to live changes in your collections and documents.
     In addition, the Database Service provides built-in validation to check the integrity of your incoming data, custom indexing for query performance,
     as well as a flexible permissions mechanism to allow you to easily segment data between different users, teams, and roles.
 </p>
 
+<h2><a href="/docs/database#database" id="database">Create Your Database</a></h2>
```

Comment:
`/docs/databases/#database`

* Reminder, Appwrite homepage repo might need updates too with renaming links
* Other places might have this link, where we need update too

Reply:
fixed